consensus/src/main/java/org/apache/iotdb/consensus/multileader/MultiLeaderConsensus.java

```diff
@@ -278,6 +278,10 @@ public ConsensusGenericResponse addPeer(ConsensusGroupId groupId, Peer peer) {
       logger.info("[MultiLeaderConsensus] activate new peer...");
       impl.activePeer(peer);
 
+      // step 7: spot clean
+      logger.info("[MultiLeaderConsensus] do spot clean...");
```

Comment:
why clean in addPeer()?

Reply:
After a peer is added, the snapshot replicas need to be transferred. After the snapshot replicas are transferred and loaded, you need to clear the temporary replica files on the remote server to prevent data space waste.